app/dashboard/templates/shared/nav_chat.html

```diff
@@ -0,0 +1,6 @@
+<div id="gc-chat" class="nav-item order-md-last">
```

Comment:
maybe just to follow the others navs we should call this file chat_nav.html

Reply:
sure